stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/typing/TypingIndicatorView.kt

```diff
@@ -2,45 +2,62 @@ package io.getstream.chat.android.ui.typing
 
 import android.content.Context
 import android.util.AttributeSet
-import android.widget.FrameLayout
+import android.view.Gravity
+import android.widget.LinearLayout
+import android.widget.TextView
 import androidx.core.view.isVisible
 import io.getstream.chat.android.client.models.User
 import io.getstream.chat.android.client.models.name
 import io.getstream.chat.android.ui.R
 import io.getstream.chat.android.ui.common.extensions.internal.createStreamThemeWrapper
+import io.getstream.chat.android.ui.common.extensions.internal.dpToPx
 import io.getstream.chat.android.ui.common.extensions.internal.streamThemeInflater
-import io.getstream.chat.android.ui.databinding.StreamUiTypingIndicatorViewBinding
 
-public class TypingIndicatorView : FrameLayout {
-
-    private val binding: StreamUiTypingIndicatorViewBinding =
-        StreamUiTypingIndicatorViewBinding.inflate(streamThemeInflater, this, true)
+public class TypingIndicatorView : LinearLayout {
 
     public constructor(context: Context) : super(context.createStreamThemeWrapper()) {
-        init()
+        init(null)
     }
 
     public constructor(context: Context, attrs: AttributeSet?) : super(context.createStreamThemeWrapper(), attrs) {
-        init()
+        init(attrs)
     }
 
     public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
         context.createStreamThemeWrapper(),
         attrs,
         defStyleAttr
     ) {
-        init()
+        init(attrs)
```

Comment:
Aren't we ignoring styles (not ours, but one create the our users) by not passing defStyleAttr along?

Reply:
🤔 good question and if yes, this bug exists in every view. Let's add a task to check that